src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -64,7 +64,7 @@ class ArrayField extends React.Component {
     return schema.additionalItems;
   }
 
-  scrollToTop() {
+  scrollToTop(callback) {
```

Comment:
Is there ever a reason for this to be something other than a `focusElement` callback? Could we simplify the call to `scrollToTop` by passing the selector?
```js
scrollToTop(focusElementSelector) {
  // ...
  if (focusElementSelector) {
    focusElement(focusElementSelector);
  }
  // ...
}

// ...
scrollToTop(`.${fieldName} .add-btn`);
```

Reply:
That's a much better solution, thanks!